### ISSUE TITLE
App window is all black after py2app

### ISSUE THREAD

**Issue by KafkaOnTheShore1981** [ronaldoussoren/py2app#276]
(1)environment:Pycharm+macOS Catalina(10.15.2)
(2)code function:adds sequence numbers as a prefix to the names of files in a selected folder. 
(3)problem: After .py code has been transformed to .app by the following steps, the window of the app goes all black although the function is ok.
Here is my procedure:
#(1)pip install py2app
#(2)py2applet --make-setup EasyReNamer.py
#(3)python setup.py py2app
then, in dist folder EasyReNamer.app is available. clicked it and the application works except for black screen.
And here is my code:
```
import os
from tkinter import filedialog
import tkinter as tk
from tkinter import Button

class ReName():
    def __init__(self):
        self.root = tk.Tk()
        self.root['background']='white'
        self.n = 0
        label = tk.Label(self.root, font=("Arial Bold", 15),text='Please select a directory to rename files in the folder:')
        self.label1=tk.Label(self.root)
        label.pack()
        self.btn = Button(self.root, font=("Arial", 15), relief='raised',highlightbackground='orange', text="Click Me", command=self.rename)
        self.btn.pack()

    def widget_position(self,root,width,height):
        self.width=self.root.winfo_screenwidth()
        self.height=self.root.winfo_screenheight()
        x=(self.width-width)/2
        y=(self.height-height)/2
        self.root.geometry('%dx%d+%d+%d' % (width,height,x,y))
        self.root.grid()

    def get_db_configure(self):
        self.root.title('Easy ReNamer Version 1.0')
        self.root.resizable(0, 0)

    def rename(self):
        file_path = filedialog.askdirectory(title='ReNamer')
        self.file_lists = os.listdir(file_path)
        for self.file in self.file_lists.copy():
            self.oldname = file_path + os.sep + self.file
            if os.path.isdir(self.oldname) or self.file.startswith('.'):
                continue
            else:
                self.newname = file_path + os.sep + '(' + str(self.n + 1) + ')' + self.file
                os.rename(self.oldname, self.newname)
                self.n+=1
        self.label1.config(text='{} file(s) renamed'.format(self.n))
        self.label1.pack()
        self.btn.config(state='disabled')
        self.btn.pack()

ins=ReName()
ins.widget_position(ins.get_db_configure(),500,80)
ins.root.mainloop()
```
Please tell me what's wrong with it and how to fix the problem. Thank you!

**Comment by Hato1** [comment]
I'm having the same problem on Mojave (10.14.6). My game using Pygame runs fine other than the display being blank (grey in light mode, black in dark mode). I believe this is bug may have been introduced with OSX Mojave and is related to https://github.com/pygame/pygame/issues/555.
Are you able to test with an older mac version?

**Comment by ronaldoussoren** [comment]
I haven't had time to try to reproduce this issue yet. Does running the application from the terminal show any errors (that is start, "EasyReNamer.app/Contents/MacOS/EasyReNamer")?

**Comment by Hato1** [comment]
No errors. I've since tested the .app that was created on my machine on an older version of mac (high sierra), ran perfectly.

**Comment by ronaldoussoren** [comment]
I can reproduce the issue on my machine, both with a regular build and with an alias build.

**Comment by ronaldoussoren** [comment]
The current tip of the tree should fix the issue (fix introduced in 56bec237e03bff4c4ce816ade8b0660f63db69a3).

Could you please test if this actually fixes the issue for you?  If it does I'll cut a release later this week.

**Comment by ronaldoussoren** [comment]
I'm pretty sure this is fixed in the repository.
